src/workerd/api/http.h

```diff
@@ -471,9 +471,15 @@ class Fetcher: public jsg::Object {
   jsg::Promise<QueueResponse> queue(
       jsg::Lock& js, kj::String queueName, kj::Array<ServiceBindingQueueMessage> messages);
 
+  jsg::Promise<kj::Maybe<jsg::V8Ref<v8::Value>>> getRpcAndCallRemote(
+      jsg::Lock& js, kj::String methodName, jsg::Optional<kj::Array<jsg::Value>> args=nullptr);
+  // Get an rpc service and call the remote method.
+  // This will evolve once we have a proxy type and can intercept method calls.
+
   JSG_RESOURCE_TYPE(Fetcher, CompatibilityFlags::Reader flags) {
     JSG_METHOD(fetch);
     JSG_METHOD(connect);
+    JSG_METHOD(getRpcAndCallRemote);
```

Comment:
This isn't the API we plan to expose publicly, right? This is just a stepping stone for testing?

Please make sure to guard this with the "experimental" compat flag so that people do not inadvertently start depending on it.

Reply:
Just a stepping stone, I'm thinking I'll get a Proxy object PR in first and then rebase this to get those changes here.